core/teverseUI/resize.lua

```diff
@@ -0,0 +1,154 @@
+math.clamp = function(x, min, max)
```

Comment:
Not sure about polluting the math global - can get a bit messy!

Reply:
Fair point, I'll push a commit to solve it.